src/main/java/com/synopsys/integration/jira/common/server/configuration/JiraServerBearerAuthRestConfig.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * int-jira-common
+ *
+ * Copyright (c) 2023 Synopsys, Inc.
+ *
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ */
+package com.synopsys.integration.jira.common.server.configuration;
+
+import java.net.URL;
+import java.util.function.BiConsumer;
+
+import com.google.gson.Gson;
+import com.synopsys.integration.jira.common.rest.JiraHttpClient;
+import com.synopsys.integration.jira.common.rest.token.JiraTokenHttpClient;
+import com.synopsys.integration.log.IntLogger;
+import com.synopsys.integration.rest.proxy.ProxyInfo;
+import com.synopsys.integration.rest.support.AuthenticationSupport;
+
+public class JiraServerBearerAuthRestConfig extends JiraServerRestConfig {
```

Comment:
Could we rename BearerAuth references to JiraServerAccessTokenRestConfig? Just for naming consistency